speech/norm/__init__.py

```diff
@@ -231,42 +182,244 @@ def _email_handler(email: str) -> str:
 _STRENGTHS = frozenset(("none", "x-weak", "weak", "medium", "strong", "x-strong"))
 
 
-def _break_handler(time: Optional[str] = None, strength: Optional[str] = None):
-    if time:
-        return f'<break time="{time}" />'
-    if strength:
-        assert strength in _STRENGTHS, f"Break strength {strength} is invalid."
-        return f'<break strength="{strength}" />'
-    return f"<break />"
-
-
-# Permissive normalization handler function type
-NormFunc = Callable[..., str]
-HANDLER_MAPTYPE = ChainMapType[str, NormFunc]
-# Default/Fallback normalization handlers,
-# voice modules can override the handlers by creating a ChainMap child
-DEFAULT_NORM_HANDLERS: HANDLER_MAPTYPE = ChainMap(
-    {
-        "number": number_to_text,
-        "numbers": numbers_to_text,
-        # ^ Plural forms are lazy shortcuts for longer text
-        # TODO: amount/s
-        # TODO: currency/ies
-        # TODO: distance/s
-        "float": float_to_text,
-        "floats": floats_to_text,
-        "ordinal": number_to_ordinal,
-        "ordinals": numbers_to_ordinal,
-        "digits": digits_to_text,
-        "phone": digits_to_text,
-        "time": _time_handler,
-        "date": _date_handler,
-        "year": year_to_text,
-        "years": years_to_text,
-        "abbrev": _abbrev_handler,
-        "email": _email_handler,
-        "break": _break_handler,
-        "paragraph": lambda txt: f"<p>{txt}</p>",
-        "sentence": lambda txt: f"<s>{txt}</s>",
+class NormalizationHandler:  # Abstract base class
+    ...
+
+
+class DefaultNormalization(NormalizationHandler):
+    """
+    Class containing default text normalization functions
+    for Icelandic speech synthesis.
+    """
+
+    # TODO
+    # amount/s
+    # currency/ies
+    # distance/s
+
+    @classmethod
+    def number(cls, txt: str, **kwargs: str):
```

Comment:
Missing return types on all of these member functions

Reply:
Fixed.